### ISSUE TITLE
Add Startup Delay Option To UI

### ISSUE THREAD

**Issue by AndrewBabbitt97** [ChromaControl/ChromaControl#105]
Add an option to change the startup delay after dependencies to the new settings UI

**Comment by hugopua** [comment]
Hi. @AndrewBabbitt97 Do you have any estimate date to add all the features on this link to the release? 
https://github.com/ChromaControl/ChromaControl/milestone/1

**Comment by AndrewBabbitt97** [comment]
@hugopua Targeting October, but might be early November.

I will be using the Windows App SDK 1.0 for the final release version. The first preview release of Windows App SDK is supposed to come out sometime this month, hopefully in the next week or two and a final stable version of it will come out around Q4 likely October or November.

I'm currently developing on Windows 11, the Windows App SDK latest version (0.8) has a breaking bug that prevents me from using it on Windows 11. All apps currently crash on startup. They should be rolling out a fix in the next week to fix this.

I finally got my new certificate from Sectigo today, so I'll be able to sign releases again starting this week. This has been a major blocker for me and I have been waiting since March for this certificate.

With this said I would like to get a new preview version out this upcoming week (dependent on when we get a fixed Windows App SDK). And am targeting a new preview version every few weeks until the stable version of the Windows App SDK around October/November. Originally the 1.0.0 release was mostly going to be similar to what's in the MSIX beta, I have decided to add the new UI to the first stable release and will be moving all the Chroma Control modules into a singular MSIX package rather than multiple.

In the future there will be two release channels, Stable and Dev for ongoing work once 1.0.0 is released.
